pytest_operator/plugin.py

```diff
@@ -387,10 +388,30 @@ def full_name(self) -> str:
         return f"{self.controller_name}:{self.model_name}"
 
 
+@dataclasses.dataclass
+class Bundle:
+    """Represents a juju bundle."""
+
+    name: str
+    channel: str = "stable"
+    arch: str = "all"
+    series: str = "all"
+
+    @property
+    def juju_download_args(self):
+        """Create cli arguments used in juju download to download bundle to disk."""
+        return [
+            f"--{field.name}={getattr(self, field.name)}"
+            for field in dataclasses.fields(Bundle)
+            if field.default is not dataclasses.MISSING
+        ]
+
+
 class OpsTest:
     """Utility class for testing Operator Charms."""
 
     _instance = None  # store instance, so we can tell if it's been used yet
+    Bundle = Bundle  # objects can be created with `ops_test.Bundle(...)`
```

Comment:
This is more a comment to think about than an actual change - I don't know the right answer

I think I understand the intent here - this is effectively giving OpsTest users the convenience function of `opstest.Bundle(...)` to create a bundle by composition, but it caught me off guard.  On first read I thought this was some `Bundle` state in a class attribute rather than an extra method on the class.  

I haven't encountered this often, so this might just be me being inexperienced with it.  It is also more elegant than the only other idea I have, which is:
```python
    @staticmethod
    def Bundle(*args, **kwargs):
      return Bundle(*args, **kwargs)
```

Reply:
Just Kidding -- `TypeAlias` is only in python 3.10

Reply:
Oh yeah are you saying that, since people just inherit the `ops_test` fixture via pytest magic, it would be odd to ask people to do something like this:
```python
from pytest-operator.plugins import Bundle

def test_something(ops_test):
  myBundle = Bundle()
  # do stuff with myBundle and ops_test ...
```

?

Yeah I have no good alternative, so unless there's someone else you can think of that might have a good suggestion I say go with it as is.

Reply:
i've got another idea i'm trying to verify with mypy

```python
class OpsTest:
    """Utility class for testing Operator Charms."""

    _instance = None  # store instance, so we can tell if it's been used yet

    # objects can be created with `ops_test.Bundle(...)`
    # since fixtures are autoloaded for pytest users,
    # importing the Bundle class isn't good UX
    # this exposes the class instantiation through
    #     ops_test.Bundle(...)
    Bundle = TypeVar["Bundle", Bundle]
```

Reply:
`mypy` didn't care for this either.  i went with the nuclear option